07_Bootstrap/praktikum/index.html

```diff
@@ -0,0 +1,166 @@
+    <nav class="navbar navbar-expand-lg bg-light">
+        <div class="container-sm">
+          <div class="col d-none d-lg-block"></div>
+          <a class="navbar-brand" href="#"><span class="text-success">e</span>Startup</a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" aria-current="page" href="#">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Screenshots</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Team</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Pricing</a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                  Dropdown
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Action</a></li>
+                  <li><a class="dropdown-item" href="#">Another action</a></li>
+                  <li><hr class="dropdown-divider"></li>
+                  <li><a class="dropdown-item" href="#">Something else here</a></li>
+                </ul>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link">Contact</a>
+              </li>
+            </ul>
+          </div>
+          <div class="col d-none d-lg-block"></div>
+        </div>
+    </nav>
+    <div class="container-fluid text-center">
+        <h1 class="text-secondary mt-2 py-3 fw-light">Symple Systeme Fordiscount</h1>
+        <p class="text-secondary pb-5 fs-6 fw-light">Integer Cursus Bibendum Augue Au Cursus</p>
+        <!--  -->
+        <div class="d-flex mt-5 justify-content-center">
+            <div class="col-3 mx-3 d-flex flex-column align-items-center">
+                <div class="col-2 d-flex flex-column">
+                    <img src="assets/img/svg/cloud.svg" alt="">
+                </div>
+                <h6 class="text-uppercase py-4 fw-bold">introducing whatsapp</h6>
+                <p class="text-secondary">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iure, ab!</p>
+                <p class="fw-semibold">Read More</p>
+            </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)